reference/cookie_consent.yml

```
@@ -0,0 +1,94 @@
+openapi: 3.0.1
+servers:
+  - url: 'https://{$$.env.store_domain}/api/storefront'
+info:
+  title: Storefront Cookie Consent API
+  description: Storefront API for specifying consent to particular categories of cookies for compliance purposes.
```

Update description to match the style of the API Reference documentation.
```suggestion
  description: Manage customer consent to particular categories of cookies for compliance purposes.
```

---

I think `Specify Shopper Cookie Consent Preferences` is better since `manage` suggests that the operation being performed is from the management perspective (i.e. user perspective) instead of the shopper perspective.